app/uk/gov/hmrc/eoricommoncomponent/frontend/views/business_details_recovery.scala.html

```diff
@@ -59,10 +62,8 @@
 }
 
 @layout_di(messages("business-details-recovery.title")) {
-<div class="column-two-thirds">
-    <a href="javascript:history.back()" id="back" class="link-back js-visible">@messages("cds.navigation.back")</a>
-
-    <h1 class="heading-large">@messages("business-details-recovery.heading")</h1>
+<div>
+    <h1 class="govuk-heading-l">@messages("business-details-recovery.heading")</h1>
 
     <dl class="govuk-check-your-answers cya-questions-long form-group">
```

Comment:
can we use Summary list without actions?

Reply:
yes again